content/ch-algorithms/teleportation.ipynb

```diff
@@ -22415,14 +22415,82 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 4. Teleportation on a Real Quantum Computer <a id='real_qc'></a>"
+    "## 4. Understanding Quantum Teleportation "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you have worked with the Quantum Teleportation's implementation, it is time to understand the mathematics behind the protocol.\n",
+    "\n",
+    "#### Step 1\n",
+    "\n",
+    "Quantum Teleportation begins with the fact that Alice needs to transmit $|q\\rangle = a|0\\rangle + b|1\\rangle$ (a random qubit) to Bob. She doesn't know the state of the qubit. For this, Alice and Bob take the help of a third party (Telamon). Telamon prepares a pair of entangled qubits for Alice and Bob. The entangled qubit could be written in Dirac Notation as,\n",
+    "$$ |\\psi \\rangle = \\frac{1}{\\sqrt{2}} (|00\\rangle + |11\\rangle) $$\n",
+    "\n",
+    "Alice and Bob each possess one qubit of the entangled pair (denoted as A and B respectively),\n",
+    "$$|\\psi\\rangle = \\frac{1}{\\sqrt{2}} (|0\\rangle_A |0\\rangle_B + |1\\rangle_A |1\\rangle_B) $$\n",
+    "\n",
+    "This creates a three qubit quantum system where Alice has the first two qubits and Bob the last one.\n",
+    "$$ \\begin{align}\n",
+    "|q\\rangle \\otimes |\\psi\\rangle &= \\frac{1}{\\sqrt{2}} (a |0\\rangle \\otimes (|00\\rangle + |11\\rangle) + b |1\\rangle \\otimes (|00\\rangle + |11\\rangle))\\\\\n",
+    "&= \\frac{1}{\\sqrt{2}} (a|000\\rangle + a|011\\rangle + b|100\\rangle + b|111\\rangle) \n",
+    "\\end{align}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 2\n",
+    "\n",
+    "Now according to the protocol Alice applies CNOT gate on her two qubits followed by Hadamard gate on the first qubit. This results in the state,\n",
+    "\n",
+    "$$ \\begin{align} (CNOT \\otimes I) (H \\otimes I \\otimes I) (|q\\rangle \\otimes |\\psi\\rangle) \\\\\n",
```

Comment:
I'm not able to see these lines on the rendered file within github (https://github.com/rochisha0/qiskit-textbook/blob/update_qt/content/ch-algorithms/teleportation.ipynb). Is something missing? The previous lines seem to be displayed okay...

Reply:
I'm not sure, everything looks fine in the local clone.